pkg/sdk/go/sdk.go

```diff
@@ -1007,6 +1024,8 @@ func (sdk mfSDK) processRequest(method, url, token, contentType string, data []b
 		return make(http.Header), []byte{}, errors.NewSDKError(err)
 	}
 
+	req.Header.Add("Referer", sdk.HostURL)
```

Comment:
Do not do it like this. Rather change `processRequest()` function in that way that we can pass any arbitrary header as a parameter.

It is not needed that we always add header, especially `Referer` header. This function should only add header when it is passed via parametr, and it should be key-value map, so that I can set any number of arbitrary headers.